fossevents/templates/base.html

```diff
@@ -15,6 +15,7 @@
       <link rel="stylesheet" type="text/css" href="{% static 'css/normalize.css' %}" />
       <link rel="stylesheet" type="text/x-scss" href="{% static 'css/main.scss' %}" />
       {% endcompress %}
+      <a href="/feed/rss/"> <img align="right" src="{% static 'images/subscribe-button.png' %}" height="70px" width="100px"> </a></img>
```

Comment:
You can't have `<a>` tag inside  `<head>`. Follow the guide from here on how to add rss meta to website: http://www.petefreitag.com/item/384.cfm